olive/systems/system_config.py

```diff
@@ -15,6 +15,16 @@
 from olive.systems.common import AzureMLDockerConfig, LocalDockerConfig, SystemType
 
 
+class VCConfig(ConfigBase):
```

Comment:
do we need add documents/comments about the meaning of these fields?

Reply:
I will send the doc link to you offline.